.github/workflows/go_tests.yml

```diff
@@ -22,3 +22,7 @@ jobs:
       run: make presubmit
     - name: Run go test
       run: make test
+    - name: Run device-injector-test
+      run: |
+        echo "${{ secrets.SUDO_PASSWORD }}" | sudo -S make device-injector-test
```

Comment:
Are we using custom actions runners?  It looks like the default GitHub Actions runners have [passwordless sudo enabled](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#administrative-privileges).

Reply:
You're right, tried sudo without the secret directly and it works. Thanks Sam!